infra/api.tf

```diff
@@ -0,0 +1,272 @@
+    file_system_id     = "string"
+    root_directory     = "string"
+    access_point_id    = "string"
+    transit_encryption = "DISABLED"
+    iam_authz          = "DISABLED"
+  }
+}
+
+data "aws_acm_certificate" "wildcard" {
+  domain      = "hotosm.org"
+  statuses    = ["ISSUED"]
+  most_recent = true
+}
+
+resource "aws_appautoscaling_target" "underpass-api" {
+  max_capacity       = 5
+  min_capacity       = 1
+  resource_id        = "service/${aws_ecs_cluster.underpass.name}/${module.ecs.service_name}"
+  scalable_dimension = "ecs:service:DesiredCount"
+  service_namespace  = "ecs"
+}
+
+resource "aws_appautoscaling_policy" "underpass-api-alb-requests" {
+  name               = "scale-by-requests-to-alb"
+  policy_type        = "TargetTrackingScaling"
+  resource_id        = aws_appautoscaling_target.underpass-api.resource_id
+  scalable_dimension = aws_appautoscaling_target.underpass-api.scalable_dimension
+  service_namespace  = aws_appautoscaling_target.underpass-api.service_namespace
+
+  target_tracking_scaling_policy_configuration {
+    scale_in_cooldown  = 120
+    scale_out_cooldown = 30
+
+    predefined_metric_specification {
+      predefined_metric_type = "ALBRequestCountPerTarget"
+      resource_label         = "app/${module.alb.load_balancer_arn_suffix}/targetgroup/${module.alb.target_group_arn_suffix}"
+    }
+
+    target_value = "5"
+  }
+}
+
+module "alb" {
+  source            = "git::https://gitlab.com/eternaltyro/terraform-aws-alb.git"
+  alb_name          = "underpass-${var.deployment_environment}"
+  target_group_name = "underpass-${var.deployment_environment}"
+  default_tags = { // TODO: move to top level var for consistency across resources
+    project       = "underpass"
+    maintainer    = "DK"
+    documentation = "https://docs.hotosm.org/"
+    cost_center   = ""
+  }
+  alb_subnets         = module.vpc.public_subnets
+  health_check_path   = "/v1/docs"
+  acm_tls_cert_domain = data.aws_acm_certificate.wildcard.domain
+  vpc_id              = module.vpc.vpc_id
+  app_port            = 8000
+  //ip_address_type
+}
+
+resource "aws_secretsmanager_secret" "configfile" {
+  name = "${var.deployment_environment}/galaxy/api-configfile"
+}
+
+resource "aws_secretsmanager_secret_version" "configfile" {
+  secret_id = aws_secretsmanager_secret.configfile.id
+  secret_string = base64encode(
+    templatefile(
+      "${path.module}/config.txt.tftpl",
+      {
+        underpass_db_creds = var.underpass_db_credentials
+
+        oauth2_creds = var.oauth2_credentials
+
+        api_url             = "${var.api_url_scheme}${var.api_host}"
+        api_port            = var.api_port
+        api_export_max_area = var.api_export_max_area
+        api_log_level       = var.api_log_level
+
+        sentry_dsn             = lookup(var.sentry, "dsn")
+        sentry_app_environment = var.deployment_environment
+        sentry_app_release_tag = "underpass-api@${var.container_image_tag}"
+      }
+    )
+  )
+}
```

Comment:
Nice. We need app releases to be done by terraform for the sentry tags to be set properly. Or we need to at least run terraform after every deployment.